node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodesV2ApiHandler.java

```diff
@@ -75,13 +76,16 @@ public class NodesV2ApiHandler extends ThreadedHttpRequestHandler {
     private final Orchestrator orchestrator;
     private final NodeRepository nodeRepository;
     private final NodeFlavors nodeFlavors;
+    private final InfraApplicationRedeployer infraApplicationRedeployer;
 
     @Inject
-    public NodesV2ApiHandler(Context parentCtx, Orchestrator orchestrator, NodeRepository nodeRepository, NodeFlavors flavors) {
+    public NodesV2ApiHandler(Context parentCtx, Orchestrator orchestrator, NodeRepository nodeRepository,
+                             NodeFlavors flavors, InfraApplicationRedeployer infraApplicationRedeployer) {
```

Comment:
You need to declare this component in services for open-source and hosted?

Reply:
Ah, yes, probably. Thanks.

Reply:
A lot simpler to just create it in the handler, then, possibly based on some other config.

Reply:
Ah, I don't think the open-source configserver comes with a node repo, does it? The system test of the PR went fine, which strongly suggests it doesn't.